setup.py

```diff
@@ -21,18 +21,51 @@
 #     Santiago Dueñas <sduenas@bitergia.com>
 #
 
+# Always prefer setuptools over distutils
```

Comment:
The import section should have the packages ordered as is described in [PEP8 document](https://www.python.org/dev/peps/pep-0008/#id23).